include/lsst/jointcal/ConstrainedPolyModel.h

```diff
@@ -46,35 +47,35 @@ class ConstrainedPolyModel : public DistortionModel
   Frame _tpFrame; // just for output of the chip transfos
 
 public :
-  ConstrainedPolyModel(const CcdImageList &L,
-		       const ProjectionHandler* ProjH,
-		       bool InitFromWCS,
-		       unsigned NNotFit=0);
+  ConstrainedPolyModel(const CcdImageList &ccdImageList,
+                       const ProjectionHandler* projectionHandler,
+                       bool initFromWCS,
+                       unsigned nNotFit=0);
 
   // The following routines are the interface to AstromFit
   //!
-  const Mapping* GetMapping(const CcdImage &) const;
+  const Mapping* getMapping(const CcdImage &) const;
```

Comment:
Is this wrapped? If so please be careful with the return value policy. Should this copy or reference_internal? Should this perhaps be smart pointer? Etc. If not wrapped then outside of scope.